aws-rds-dbinstance/src/test/java/software/amazon/rds/dbinstance/client/RdsClientProviderTest.java

```diff
@@ -90,8 +91,8 @@ public void test_getClientWithApiVersion_fullExec() throws IOException {
         byte[] requestBytes = IOUtils.toByteArray(httpExecuteRequest.contentStreamProvider().get().newStream());
         final String requestBody = new String(requestBytes);
 
-        // Ensure the client encoded {@code apiVersion} into the request POST body.
-        Assertions.assertThat(requestBody).contains("Version=" + apiVersion);
+        //TODO: Ensure the client encoded {@code apiVersion} into the request POST body.
```

Comment:
Please prepare a PR for this after releasing GP3 support.

Reply:
Will do.